tracee-ebpf/Makefile

```diff
@@ -132,7 +127,6 @@ $(OUT_DIR)/tracee.bpf.core.%.o: $(BPF_SRC) $(LIBBPF_HEADERS) $(CMD_CLANG)
 		-DLINUX_VERSION_CODE=$(LINUX_VERSION_CODE) \
 		-target bpf \
 		-O2 -c -g -o $@ $<
-endif
 else
```

Comment:
DOCKER mode seem to be inconsistently supported for the obj files now. We need to be able to build both obj targets with docker and without. currently I think it's not possible? (haven't tried, just by reading the code)
If I'm right, would be easier to implement, and also more readable to just copy the same stanza from the other targets (e.g the OUT_BIN target) where the DOKCER variance is inside the target, not outside.

Reply:
is it resolved? I don't see any change in the code

Reply:
Ugh, I forgot to git push...